app/src/main/java/com/example/farmeraid/home/views/HomeScreenView.kt

```diff
@@ -47,6 +48,10 @@ fun HomeScreenView() {
     val viewModel = hiltViewModel<HomeViewModel>()
     val state by viewModel.state.collectAsState()
 
+    LaunchedEffect(Unit) {
+        viewModel.fetchData()
+    }
```

Comment:
Just a note that this makes it so that everytime we come back to the HomeScreen page, it refetches data. Before it didn't refetch on navigate back. I currently have it doing it only for the home screen as the home screen is like a hub overviewing everythign and most people would expect it to always be updated when they navigate back.

Reply:
Perfect!  I was wondering how we were going to address this when we were doing changes to quotas this is great!